lib/server.coffee

```diff
@@ -5,6 +5,18 @@ crypto = Npm.require 'crypto'
 
 cacheCollection = new Mongo.Collection 'SpiderableCacheCollection'
 
+hashCode = (string) ->
```

Comment:
Based on top of charcode - should be unique as Meteor's mono id (which at some point might be not unique, but chance is dramatically low)